devimint/src/gatewayd.rs

```diff
@@ -228,6 +229,25 @@ impl Gatewayd {
         Ok(address)
     }
 
+    pub async fn get_mnemonic(&self) -> Result<MnemonicResponse> {
+        let value = retry(
+            "Getting gateway mnemonic",
+            backoff_util::aggressive_backoff(),
+            || async { cmd!(self, "seed").out_json().await },
+        )
+        .await
+        .context("Getting gateway mnemonic")?;
```

Comment:
Can you explain why this needs to be wrapped in a `retry()` call?

Reply:
In the test, this function is used right after restarting/upgrading the gateway. To avoid flakiness (i.e requesting the mnemonic when the webserver isnt up yet), it is easiest to just wrap in a retry.